exercises/two-fer/src/test/kotlin/TwoferTest.kt

```diff
@@ -0,0 +1,24 @@
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.Assert.assertEquals
```

Comment:
Just a note that https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-equals.html is probably available - I assume it's a wrapper around the JUnit assertion anyway though!